src/frontend/admin/src/utils.js

```diff
@@ -14,11 +14,11 @@ export const useLocalStorageState = (key, defaultValue) => {
 };
 
 export const createApiPost = (apiUrl) =>
-    function (path, body = {}, method = "POST") {
+    function (path, body = {}, method = "POST", sendRaw = false) {
```

Comment:
At one point in time we might have a huge number of parameters for this function. 
I think there are two nice solutions to keeping the code nice and tidy:
1) Use 3rd party library like Axios
2) Move all parameters to a single object and invoke the function like this:
```javascript
apiPost("", constn, "POST", {
sendRaw: true
});
```
That way we can have more parameters that are not required